Formula/gnuradio.rb

```diff
@@ -145,14 +151,19 @@ def install
       -DENABLE_INTERNAL_VOLK=OFF
     ]
 
-    enabled = %w[GNURADIO_RUNTIME GR_ANALOG GR_AUDIO GR_BLOCKS GRC
```

Comment:
Why do we want to separate this into `enabled` and `enabled_modules`?

Reply:
GR_ modules are optional, and the other enable flags seem to be necessary

Reply:
btw, maybe it's a good idea to have another formula like `gnuradio-full` with all the optional modules. This would be way easier than adding new modules here everytime some needs a specific one

Reply:
No, that's not something we're interested in doing.

We try to package software so that it fills the most common use cases. That means enabling all the most useful features, and perhaps also ones that aren't widely used but are nevertheless not too costly to add to the formulae (e.g. it doesn't add heavy dependencies, and doesn't add too much to the build time/bottle size).

If users are looking for more flexibility than that, we encourage them to host a customised version of the formula in an external tap. See, for example: https://docs.brew.sh/Interesting-Taps-and-Forks#unsupported-interesting-taps